### PR TITLE
VS Team service actions to the exception details page.

### PR DESCRIPTION
**Pull request by kshyju**
Added Visual studio team services to the exception details page. Similar to JIRA actions, User can create a bug or a pbi from this page. I also changed the javascript (link click event handler) to be generic so that it works for both jira actions and tfs action now.

**Edit** : I pushed few more commits ( _I did not know it was going to be part of this PR!_) after my initial PR for the VS Team services integration. The commits are for

1) Enabled the "Views" tab on the database screen.
2) Enabled the "Missing Indexes" tab on the database scree (for #122)

Please review. Thank you :)

### CODE REVIEW

**Comment by NickCraver** [comment]
This PR is going against `master` and not the active `overhaul` dev branch - so I can't accept it without causing a lot of grief to all commits on both sides.

General issues: I see a ton of whitespace changes which makes any PR really hard to read. Can you please try and avoid this in the future?

On the specific bits: 
1. TFS Actions is sharing the wrong bits of code with Jira, but honestly they shouldn't really co-exist like this anyway - I want to revisit this when a plugin model is good to go as this should be a very simply plugin you opt into (rather than the inevitable 40 `if`s in a view we'd currently have with all the bug trackers). A few things I want to get into such a plugin: 

-AD auth using the service account we're running as.
-The ability to select which project (which means we're smart about TFS and fetch some data from it)
TL;DR: I want to hold off on integration like this until we can do it proper, and likely not in the core codebase. This opens up options like using TFS libs out there. Doing the above now would create active blockers to the .Net Core port I don't want to add in the way.

2. SQL Views: Should be a different PR, and is missing many critical bits - most of this is in a quick version over in `overhaul` already, but I didn't get to definition (which will be on-expansion) which I'll try and do tonight. It'll have pretty print, etc. - I just ran out of time before my trip :)

3. SQL Missing Indexes: Again, same issues as above, but I had something much more useful in mind here - best shown in code. I'll leave this open and post a comment when it's gotten to hopefully this week. Disclaimer: I have a hardware blog post to get out about Stack Overflow in the same timeframe, so it may be another week because of the blog.

**Comment by kshyju** [comment]
So you have 2 & 3 being developed in the overhaul branch. rite ? In that case, should i close this PR ?

**Comment by NickCraver** [comment]
Yep: 2 is in there (I'll add definition today, didn't get to it last night...fixed a few other issues though), 3 needs more work and is local, it'll show up in overhaul on github likely next week.

**Comment by kshyju** [comment]
Thanks! How do we know which items you are /are not working on so that this thing(duplicate effort) won't happen again.  The missing index item in issue list did not have any info that someone was working on that.

**Comment by NickCraver** [comment]
There's not a formal list, since I'm just working as much as I can across all projects as time allows (time is my most constrained resource). I'd suggest opening an issue here or commenting on an existing one before spending time on code. I have a lot of changes planned on the plugin front and changes intended to go towards an upcoming ASP.Net MVC 6/.Net Core port.

I concede this is far from an ideal situation, but in most of the cases here I'd spend more time explaining and redoing than just coding it (especially on .Net Core-related items). So for the current push to 2.0, I'm just trying to knock out as much as I can and get this back into `master` for everyone.

**Comment by NickCraver** [comment]
@kshyju I added prettified view definitions to overhaul in https://github.com/opserver/Opserver/commit/906225486e6ac0f59f3d22e2990f0402f1846e3f - here's a screenshot of what they look like: 
screen shot 2016-03-06 at 2 37 33 pm

**Comment by kshyju** [comment]
I like the overhaul! Nice work.  loves the new progess bar images :+1:

**Comment by kshyju** [comment]
@NickCraver  Did the missing indexes view make it to the overhaul yet ?

**Comment by kshyju** [comment]
Closing.
